source-jira/source_jira/__main__.py

```diff
@@ -0,0 +1,24 @@
+import estuary_cdk.pydantic_polyfill  # Must be first.
+
+import asyncio
+from estuary_cdk import shim_airbyte_cdk
+from source_jira import SourceJira
+
+
+def wrap_with_braces(body: str, count: int):
+    opening = "{" * count
+    closing = "}" * count
+    return f"{opening}{body}{closing}"
+
+
+def urlencode_field(field: str):
```

Comment:
Do we need `urlencode_field` and `wrap_with_braces`? I don't see them being used anywhere.